packages/flutter/lib/src/widgets/form.dart

```diff
@@ -256,6 +257,9 @@ class FormField<T> extends StatefulWidget {
   /// autovalidates, this value will be ignored.
   final bool autovalidate;
 
+  /// If true, indicates that the widget is enabled.
+  final bool enabled;
```

Comment:
docs should be more detailed. What does it mean to be enabled? What does it mean to be disabled? For example, can you still select text when it's disabled? Is validation affected?

Reply:
The comment has been expanded to provide more information